### ISSUE TITLE
It seems like null checks are not enforced in millisecond code

### ISSUE THREAD

**Issue by michaelhidalgo** [unshiftio/millisecond#5]
Hi there,

I found an issue in the code because null checks are not enforced. Namely the code assumes that, match object from the RegEx,  will contain an element at the position 3 (or match[2] ) which can be null and therefore throws an exception when trying to read a property from null.

This test verifies the problem : 

```javascript
  it('should enforce null checks /(null checks not enforced.)', function(){
    assume(function () { ms("000") }).to.throw('Cannot read property \'toLowerCase\' of undefined');
  });
```

and the fix is to add a null check in line 55 : 

``` if (match[2]== undefined) return 0;```

In CoffeeScript it would be even easier since the existential operator "?" which would make the code easier to write.

A more complete (and better solution would be) 

``` if (match[2] == null) {
    return 0;
  }```

**Comment by michaelhidalgo** [comment]
I've sent a pull request with a fix, all test are running https://github.com/unshiftio/millisecond/pull/6.

**Comment by lpinca** [comment]
Fixed in 77e92c4.
